src/miner.cpp

```diff
@@ -855,6 +1041,73 @@ void StakeMiner(CWallet *pwallet)
     RenameThread("grc-stake-miner");
 
     MinerAutoUnlockFeature(pwallet);
+    
+    // Parse StakeSplit and SideStaking flags.
+    bool fEnableStakeSplit = GetBoolArg("-enablestakesplit");
+    LogPrintf("StakeMiner: fEnableStakeSplit = %u", fEnableStakeSplit);
+
+    bool fEnableSideStaking = GetBoolArg("-enablesidestaking");
+    LogPrintf("StakeMiner: fEnableSideStaking = %u", fEnableSideStaking);
+
+    vector<string> vSubParam;
+    SideStakeAlloc vSideStakeAlloc;
+    std::string sAddress;
+    int64_t nMinStakeSplitValue;
+    double dEfficiency;
+    double dAllocation;
+    
+    // If side staking is enabled, parse destinations and allocations.
+    if (fEnableSideStaking)
+    {
+        if (mapArgs.count("-sidestake") && mapMultiArgs["-sidestake"].size() > 0)
+        {
+            for (auto const& sSubParam : mapMultiArgs["-sidestake"])
+            {
+                ParseString(sSubParam, ',', vSubParam);
+                sAddress = vSubParam[0];
+                
+                try
```

Comment:
I have concern here. with the try failures; It will continue the loop and continue allocations. 

With a case where one is a bad allocation; is there a fall back plan so that the remainder of the side stake split that has not been allocated is not lost or not used? not my area of expertise but thought id ask what would happen if there was a failure of an allocation.